README.md

```diff
@@ -39,26 +39,31 @@ provisioning VMs, and [OpenShift Origin](https://github.com/openshift/origin) fo
 <a name="prerequisites"></a>
 ### Prerequisites
 
-Minishift requires a hypervisor to run the virtual machine containing OpenShift. Depending on your
-host OS, you have the choice of the following hypervisors:
+Minishift requires a hypervisor to start the virtual machine containing the single-node OpenShift cluster.
```

Comment:
It does not contain the single node OpenShift cluster, but the cluster gets created during the start. We should fix the sentence to reflect this.